Automattic/jetpack
# Protect: different copy for unchecked items

#### Changes proposed in this Pull Request:

* Use a different copy when there are no found vulnerabilities but there are still some unchecked items

#### Jetpack product discussion


p9dueE-51o-p2#comment-7864

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate and set up Protect
* use `complete_green` response type and confirm you see the "Dont worry about a thing message"
* change to `incomplete_green`response type and confirm you see the new copy, as per the image below

![Captura de tela de 2022-05-05 13-52-17](https://user-images.githubusercontent.com/971483/166973459-de25333c-64a7-4884-87d3-4639c1209abb.png)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202183900595802
